tabs-tabs-tabs/tabs.js

```diff
@@ -12,6 +12,30 @@ function firstUnpinnedTab(tabs) {
   }
 }
 
+function listTabs() {
+  chrome.tabs.query({
+    currentWindow: true
+  }, function(tabs) {
+    var tabsList = document.getElementById('tabs-list'),
+        currentTabs = document.createDocumentFragment();
+
+    tabsList.innerHTML = '';
+
+    for (var tab of tabs) {
+      if (!tab.active) {
+        var tabLink = document.createElement('a');
+
+        tabLink.innerHTML = tab.title || tab.id;
+        tabLink.setAttribute('href', tab.id);
+        tabLink.classList.add('switch-tabs');
+        currentTabs.appendChild(tabLink);
+      }
+    }
+
+    tabsList.appendChild(currentTabs);
```

Comment:
This list is going to get huge if you've got like 100 tabs open. People do wierd things like that, I wonder if we should limit this down to just a few to stop something bad happening. Once we've demonstrated the point on a few tabs, there's no need to do it for them all. 

We did this on history-deleter for a similar reason.